main.py

```diff
@@ -13,5 +13,5 @@
  
 #print client.order_create('AUD', 'LTC', 100000000, 100000000, 'Bid', 'Limit', '1')
 
-#print client.account_balance()
+print client.get_tick('ETH','AUD')

```

Comment:
good work. Please only change this to "get_market_tick" instead of "get_tick" so your sample also works. This can be merged afterwards.  thanks.